scripts/kconfig/kconfigfunctions.py

```diff
@@ -6,10 +6,20 @@
 import sys
 import os
 import re
+sys.path.append(os.environ['srctree'])
+from scripts.dts.devicetree import parse_file
+from scripts.dts.extract.globals import get_reduced, reduced
```

Comment:
This would be a more standard way to do it:

```python
sys.path.insert(0, os.path.join(os.environ['srctree'], 'scripts/dts'))
from devicetree import parse_file
from extract.globals import get_reduced, reduced
```

Usually `import foo.bar.baz` implies that `foo` and `bar` are Python packages (directories with an `__init__.py` file in them), though it might work to think of `.` as just a path separator in Python 3. Think it's due to this obscure thing: https://www.python.org/dev/peps/pep-0420/